actors/builtin/miner/monies.go

```diff
@@ -211,15 +211,25 @@ func LockedRewardFromReward(reward abi.TokenAmount) (abi.TokenAmount, *VestSpec)
 }
 
 var EstimatedSingleProofGasUsage = big.NewInt(65733297) // PARAM_SPEC
-var BatchDiscount = builtin.BigFrac{                    // PARAM_SPEC
+var EstimatedSingleProveCommitGasUsage = big.NewInt(49299973)
+var EstimatedSinglePreCommitGasUsage = big.NewInt(16433324)
+var BatchDiscount = builtin.BigFrac{ // PARAM_SPEC
 	Numerator:   big.NewInt(1),
 	Denominator: big.NewInt(20),
 }
-var BatchBalancer = big.Mul(big.NewInt(2), builtin.OneNanoFIL) // PARAM_SPEC
+var BatchBalancer = big.Mul(big.NewInt(5), builtin.OneNanoFIL) // PARAM_SPEC
 
-func AggregateNetworkFee(aggregateSize int, baseFee abi.TokenAmount) abi.TokenAmount {
+func AggregateProveCommitNetworkFee(aggregateSize int, baseFee abi.TokenAmount) abi.TokenAmount {
+	return AggregateNetworkFee(aggregateSize, EstimatedSingleProveCommitGasUsage, baseFee)
+}
+
+func AggregatePreCommitNetworkFee(aggregateSize int, baseFee abi.TokenAmount) abi.TokenAmount {
+	return AggregateNetworkFee(aggregateSize, EstimatedSinglePreCommitGasUsage, baseFee)
+}
+
+func AggregateNetworkFee(aggregateSize int, gasUsage big.Int, baseFee abi.TokenAmount) abi.TokenAmount {
```

Comment:
Can we not export this? I don't think it's needed, and it's only too easy for something (Lotus) to misuse this since it shares a name with the v5 method

Reply:
no problem